src/res/apps.yaml

```diff
@@ -32,22 +32,22 @@ items:
       - https://blog.osscameroon.com
       - https://miniyotas.osscameroon.com
     image: default.svg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```

Comment:
It looks like your text editor, has reformatted the file, is it possible to change this line back, as they were before ?

Reply:
Alright, thanks for pointing that. I did not notice. please help me interpret what changed?
what does the blue and red represent?

Reply:
Sure, The green line refers to what you have added in your PR, the stronger green on line `35` highlight a space between the beginning of the line and the comment. You should remove this space, I guess